cpp/src/parquet/printer.cc

```diff
@@ -166,10 +166,20 @@ void ParquetFilePrinter::DebugPrint(std::ostream& stream, std::list<int> selecte
       stream << "  Values: " << column_chunk->num_values();
       if (column_chunk->is_stats_set()) {
         std::string min = stats->min(), max = stats->max();
+        std::string max_exact =
+            stats->is_max_value_exact.has_value()
```

Comment:
Is it an overkill to print `unknown`? Could we simplify it as below:

```
std::string max_exact = stats->is_max_value_exact.value_or(false) ? "true" : "false";
```

Reply:
To be honest I do prefer to print `unknown`. If I am doing some debugging/investigation I'd rather see that it's unknown instead of false as that wouldn't be really correct and might produce some misinterpretation but I am happy to do what the majority prefers :)